### ISSUE TITLE
Reenable MIPS, PPC, s309x builder images from go1.13, go1.14

### ISSUE THREAD

**Issue by kvch** [elastic/golang-crossbuild#71]
Right now `make build` fails with the following error for the MIPS builder images.

```
Step 7/13 : RUN     apt-get install -qq -y       --no-install-recommends       --allow-unauthenticated         crossbuild-essential-mipsel         gcc-mips-li
nux-gnu         g++-mips-linux-gnu         gcc-6-mips64-linux-gnuabi64         g++-6-mips64-linux-gnuabi64         gcc-6-mips64el-linux-gnuabi64         g++-6
-mips64el-linux-gnuabi64         librpm-dev:mips64el         libc-dev:mips64el         libpopt-dev:mips64el         linux-libc-dev:mips64el         libxml2-de
v:mips64el         libxml2:mips64el         libicu-dev:mips64el         libicu57:mips64el         icu-devtools:mips64el         libsystemd-dev:mips64el     &&
 rm -rf /var/lib/apt/lists/*
 ---> Running in 612d6e75c359
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c apt-get install -qq -y       --no-install-recommends       --allow-unauthenticated         crossbuild-essential-mipsel         gcc-mip
s-linux-gnu         g++-mips-linux-gnu         gcc-6-mips64-linux-gnuabi64         g++-6-mips64-linux-gnuabi64         gcc-6-mips64el-linux-gnuabi64         g
++-6-mips64el-linux-gnuabi64         librpm-dev:mips64el         libc-dev:mips64el         libpopt-dev:mips64el         linux-libc-dev:mips64el         libxml
2-dev:mips64el         libxml2:mips64el         libicu-dev:mips64el         libicu57:mips64el         icu-devtools:mips64el         libsystemd-dev:mips64el
  && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
make[2]: *** [../Makefile.common:15: build] Error 100
```

As we are not using this image anywhere I have disabled it temporarily in #70.

I also had to disable `ppc` and `s309x` platforms.

**Comment by kuisathaverat** [comment]
we are still waiting for a decision to close this both PRs achieve the needs and resolve the current issue.

https://github.com/elastic/golang-crossbuild/pull/62
https://github.com/elastic/golang-crossbuild/pull/65

**Comment by fearful-symmetry** [comment]
So, it sounds like in theory this will be fixed by #62?

**Comment by kuisathaverat** [comment]
the Go 1.15.6 Docker images has the issue resolved
